### DISCUSSION TITLE
TensorRT extension from A1111

### DISCUSSION THREAD

**Discussion by ghost** [vladmandic/sdnext#1184] [General]
https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/10684#discussioncomment-6019118

My dreams come true. I did not try it out yet but this looks promising. I hope it is compatible with SD.Next? I think it is

**Comment by vladmandic** [comment]
created #1185 to track it.

**Comment by cool-bigdogs-tshirt** [comment]
PR here with the necessary changes to this repo if anyone else wants to test: https://github.com/vladmandic/automatic/pull/1199

i'm having trouble getting trtexec to work myself
